src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -31,7 +31,8 @@ const DomainList = (props) => {
     onPressBody,
     DefaultImage,
     isCommunity,
-    isBlockedSection
+    isBlockedSection,
+    withKarma
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding the `withKarma` prop to the destructuring of props is necessary for the conditional rendering of the Karma score in the `ProfilePicture` component. Good addition.

```diff
-    isBlockedSection
+    isBlockedSection,
+    withKarma
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->